<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GdiPaintClass</class>
 <widget class="QWidget" name="GdiPaintClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1115</width>
    <height>746</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GdiPaint</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3" columnstretch="1,1,4,1,3,1,1,1">
   <item row="1" column="0">
    <widget class="QPushButton" name="pushButton_open">
     <property name="text">
      <string>open</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pushButton_quadri">
     <property name="text">
      <string>quadri</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>450</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="pushButton_test">
     <property name="text">
      <string>switch</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="5">
    <widget class="QPushButton" name="pushButton_red">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>red</string>
     </property>
    </widget>
   </item>
   <item row="1" column="6">
    <widget class="QPushButton" name="pushButton_green">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>green</string>
     </property>
    </widget>
   </item>
   <item row="1" column="7">
    <widget class="QPushButton" name="pushButton_blue">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>blue</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="8">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="ROIView" name="viewer" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="MFCpaint" name="MFCcanvas" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ROIView</class>
   <extends>QWidget</extends>
   <header>roiview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MFCpaint</class>
   <extends>QWidget</extends>
   <header>mfcpaint.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="GdiPaint.qrc"/>
 </resources>
 <connections/>
</ui>
